opta-mern-stack/awsenv.yaml

```diff
@@ -0,0 +1,10 @@
+name: awsenv
+org_name: runx
+providers:
+  aws:
+    region: us-east-1
+    account_id: 652824312345
```

Comment:
Its a fake account_id.

Reply:
we can do what JD has done here https://github.com/run-x/opta-examples/blob/main/mattermost-team/mattermost-aws.yaml#L5-L6
so Opta would fail right away if the user forgot to set these values